.circleci/config.yml

```diff
@@ -1030,9 +975,83 @@ workflows:
           path: release
           filters:
             branches:
-              ignore: /.*/
+              ignore: /.*/     # so far not working, so do not run (revisit at some point)
             tags:
-              only: /^v?\d+\.\d+\.\d+(?:-.*)?$/
+              ignore: /.*/     # so far not working, so do not run (revisit at some point)
           requires:
             - build-linux
             - build-mac
+
+  # run this workflow for all branches apart those reserved for chart
+  build-docker-image:
+    jobs:
+      - build-docker-image:
```

Comment:
So if the "testnet" branch is built using this action, we're not pushing to docker hub

Reply:
Nevermind. My bad.